integer/src/ibig.rs

```diff
@@ -68,6 +70,23 @@ use crate::{
 pub struct IBig(pub(crate) Repr);
 
 impl IBig {
+    #[inline]
```

Comment:
Are you aware of the [DivRem](https://docs.rs/dashu-base/latest/dashu_base/ring/trait.DivRem.html) and [DivRemEuclid](https://docs.rs/dashu-base/latest/dashu_base/ring/trait.DivRemEuclid.html) trait provided in the `dashu-base` crate?